install.py

```diff
@@ -1,4 +1,14 @@
 if __name__ == "__main__":
-    from sd_dynamic_prompts.version_tools import install_requirements
+    try:
+        from sd_dynamic_prompts.version_tools import install_reequirements  # noqa: F401
```

Comment:
typo 
should be `install_requirements` not `install_reequirements`

Reply:
Ah, yeah, oops, I left that in accidentally while testing this. :D